.travis.yml

```diff
@@ -25,7 +26,7 @@ env:
     # LATEST_RELEASE_REPO is the dockerhub repo that FLOATING_REPO should
```

Comment:
Should really change the FLOATING_REPO comment to match, otherwise it could be a bit confusing.